templates/prometheus.go

```diff
@@ -155,6 +155,10 @@ var PrometheusTemplate = promv1.Prometheus{
 						Regex:        getRelableRegex(alerts, metrics),
 						Action:       "keep",
 					},
+					{
+						Regex:  "^documentation$",
+						Action: "labeldrop",
+					},
```

Comment:
This relabel will only affect remote writes, we need to drop the documentation from the time series inside our prom to prevent the doc links from being visible in the OCS local dashboard

Reply:
Right, relabelling won't work in that case. Should I revert back to the earlier commit?

Reply:
Why not relabel upon scraping? I believe we did it somewhere else as well?
@ezio-auditore Can you help here?

Reply:
I'll make the changes based on [this](https://grafana.com/blog/2022/03/21/how-relabeling-in-prometheus-works/).

```
First off, the relabel_configs key can be found as part of a scrape job definition. These relabeling steps are applied before the scrape occurs and only have access to labels added by Prometheus’ Service Discovery. They allow us to filter the targets returned by our SD mechanism, as well as manipulate the labels it sets.

Once the targets have been defined, the metric_relabel_configs steps are applied after the scrape and allow us to select which series we would like to ingest into Prometheus’ storage.

Finally, the write_relabel_configs block applies relabeling rules to the data just before it’s sent to a remote endpoint. This can be used to filter metrics with [high cardinality](https://grafana.com/blog/2022/02/28/how-to-manage-cardinality-with-out-of-the-box-dashboards-in-grafana-cloud/?pg=blog&plcmt=body-txt) or route metrics to specific remote_write targets.
```

Reply:
yes @rexagod you will need to relable while it is being scraped.Otherwise it is only going to affect the remote write data.